app/Jobs/Service/Install/VerifyInstallation.php

```
@@ -38,175 +38,188 @@ public function handle(): void
+                        'storage'
+                    ],
+                    // Vérifier que l'application répond
+                    'http_check' => true
                 ];
 
-                $result = Process::timeout(30)->run($checkFileCommand);
+                // 1. Vérification des fichiers essentiels
+                foreach ($verifications['files'] as $file) {
+                    // Construction de la commande SSH avec Process
+                    $sshPassword = config('batistack.ssh.password') ?? env('SSH_PASSWORD');
+                    $sshCommand = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sshCommand'. (undefined)

(UnusedLocalVariable)